app/api/business/validators/application_validator.py

```diff
@@ -212,12 +212,46 @@ def validate_methods(self):
 
     def validate_recruiter(self):
         errors = []
-        if not self.application.data.get('recruiter'):
+        recruiter = self.application.data.get('recruiter')
+
+        if not recruiter:
             errors.append({
                 'message': 'Recruiter is required',
                 'severity': 'error',
                 'step': 'recruiter'
             })
+
+        if recruiter == 'yes' or recruiter == 'both':
+            labour_hire = self.application.data.get('labourHire', {})
+            now = pendulum.now('Australia/Canberra').date()
+            for state, state_value in labour_hire.iteritems():
+                if not state_value:
+                    continue
+                licence_number = state_value.get('licenceNumber')
+                expiry = state_value.get('expiry')
+                if (licence_number and not expiry) or (not licence_number and expiry):
```

Comment:
```suggestion
                if (licence_number and not expiry) or (not licence_number and expiry):
```
if not licence_number or not expiry: